test/temporal.jl

```diff
@@ -53,10 +42,19 @@
 
             @testset "dims = 1" begin
                 d = 1
-                expected = [[x] for x in [9:23..., 0, 1, 3:9...]]
-                @test Transforms.apply(x, hod; dims=d, inds=[1]) == expected
+                @test Transforms.apply(x, hod; dims=d, inds=[2, 3, 4, 5]) == expected
             end
         end
+
+        @testset "DST" begin
+            x = ZonedDateTime(2020, 3, 7, 9, 0, tz"America/New_York"):Hour(1):ZonedDateTime(2020, 3, 8, 9, 0, tz"America/New_York")
+
+            # expected result skips the DST transition hour of 2
+            expected = [9:23..., 0, 1, 3:9...]
```

Comment:
```suggestion
            expected_dst= [9:23..., 0, 1, 3:9...]
```
only because it could get interpreted that we've reassigned the `expected` used in a bunch of other tests.